### PR TITLE
Fix 5.1 Repetition Codes typo

### PR DESCRIPTION
**Pull request by ivanightingale**
## What Changes Were Made?
Fix a few typos and clarify a few sentences.

### CODE REVIEW

File: content/ch-quantum-hardware/error-correction-repetition-code.ipynb [hunk 1]
```diff
@@ -7031,7 +7031,7 @@
    "source": [
     "The resulting logical error probabilities are displayed in the following graph, whch uses a log scale used on the y axis. We would expect that the logical error probability decays exponentially with increasing $n$. If this is the case, it is a confirmation that the device is compatible with this basis test of quantum error correction. If not, it implies that the qubits and gates are not sufficiently reliable.\n",
     "\n",
-    "Fortunately, the results from IBM Q prototype devices typically do show the expected exponential decay. For the results below, we can see that small codes do represent an exception to this rule. Other deviations can also be expected, such as when the increasing the size of the code means uses a group of qubits with either exceptionally low or high noise."
+    "Fortunately, the results from IBM Q prototype devices typically do show the expected exponential decay. For the results below, we can see that small codes do represent an exception to this rule. Other deviations can also be expected, such as when an increase in the size of the code introduces a group of qubits with either exceptionally low or high noise."
```
Comment (HwajungKang) [inline content/ch-quantum-hardware/error-correction-repetition-code.ipynb:7034]: I am not sure if "introduce" is the right word here. I would rather say "could include".
But I want @quantumjim to check this.

Comment (quantumjim) [reply content/ch-quantum-hardware/error-correction-repetition-code.ipynb:7034]: "means using" is what I originally intended. "introduces" is not so bad, but perhaps "could include" is better.

Comment (ivanightingale) [reply content/ch-quantum-hardware/error-correction-repetition-code.ipynb]: @quantumjim changed it. Maybe "such as when xxx includes qubits that..." sounds more right than "such as when xxx could include qubits that..."? Either way, I think the sentence is probably clearer than before.

**Comment by ivanightingale** [comment]
@HwajungKang Hi, I don't have write access to merge. Could you please help me merge the pr?

**Comment by HwajungKang** [comment]
@ivanightingale I see there are conflicts that need to be resolved first.

**Pull request merged**